packages/amplify-frontend-javascript/lib/framework-config-mapping.js

```diff
@@ -37,6 +37,13 @@ const vueConfig = {
   StartCommand: `${npm} run-script serve`,
 };
 
+const emberConfig = {
+  SourceDir: '',
```

Comment:
Shouldn't the SourceDir be src or `/`?

Reply:
I guess `/` felt a bit too much like "root of the file system" to me 😂 and the fact that the only one that was using it was ReactNative made me feel like it wasn't what I needed to use 😖 

I tested with `''` and it worked fine, I'm testing now with `'/'` and I'll push the change if it works 👍

Reply:
Thank you @mansona!

Reply:
@kaustavghosh06 I have pushed that now (force pushed to keep history clean)